### PR TITLE
Wolo 286 reserve list

### PR DESCRIPTION
**Pull request by ayakiri**
Jira: https://woloapp.atlassian.net/browse/WOLO-286

- joiniing reserve (event http://localhost:3000/details/14 is easy to fill)
- leaving reserve

### CODE REVIEW

File: src/Views/Details/ShiftCard/ShiftCard.js [hunk 1]
```diff
@@ -21,36 +22,45 @@ const ShiftCard = ({ shift, city, isInPast }) => {
...
     const isModerator = roles && roles.includes('MODERATOR');
     const isAdmin = roles && roles.includes('ADMIN');
-    const isFull = shift.registeredUsers === shift.capacity;
+    const isFull = shift.registeredUsers >= shift.capacity;
     const [userConfirmed, setUserConfirmed] = useState(false);
 
     const [confirmPhone, setConfirmPhone] = useState(false);
     const [confirmLeave, setConfirmLeave] = useState(false);
 
+    const isSignedIn = userShifts.map(shift => shift.shiftId).includes(shiftId);
```
Comment (KinWaj) [inline src/Views/Details/ShiftCard/ShiftCard.js:34]: image
I don't know what's going on but I can't launch any event due to that
Did even `npm install`, didn't work, I can't see this issue when I'm not logged in, something to do with logged in users?

**Review by KinWaj — COMMENTED** [review]
As admin, I cannot look at the event. It gives me errors.
image

**Review by KinWaj — CHANGES_REQUESTED** [review]
need fix for admin, other than that works

**Review by kamil-kaczor — APPROVED** [review]

**Review by KinWaj — APPROVED** [review]
works

**Pull request merged**